PosterCalibrationSample/Unity/Assets/SampleAssets/AniCrane/AnimationSpeed.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections;
+
+public class AnimationSpeed : MonoBehaviour
+{
+    public float speed = 1.0f;
+
+    // Use this for initialization
+    void Start()
+    {
+        foreach (AnimationState state in this.gameObject.GetComponent<Animation>())
+        {
+            state.speed = speed;
```

Comment:
Is there a reason to do this instead of changing the speed of the source asset? I haven't seen the implementation, but I'm just curious if this is a workaround from changing the animation when it may be used without this script and not work as intended.